zebra-rpc/src/methods.rs

```diff
@@ -35,10 +51,15 @@ pub trait Rpc {
 pub struct RpcImpl;
 impl Rpc for RpcImpl {
     fn get_info(&self) -> Result<GetInfo> {
-        // TODO: dummy output data, fix in the context of #3142
         let response = GetInfo {
-            build: "TODO: Zebra v1.0.0 ...".into(),
-            subversion: "TODO: /Zebra:1.0.0-beta.../".into(),
+            build: String::from_utf8(
```

Comment:
Suggestion (3/3): return a copy of the string from the RPC method object

```suggestion
            build: self.app_version.clone(),
```